proxy/pom.xml

```diff
@@ -19,32 +19,19 @@
 
   <modules>
     <module>api</module>
-    <module>test</module>
```

Comment:
why?

Reply:
You didn't miss it before, I introduce it now. Sorry for the confusion.

Reply:
> sorry, do it like this:
>
> https://github.com/smallrye/smallrye-health/blob/main/pom.xml#L260-L271
>
> basically if not performing release, not if not skipping tests.

I would like to avoid introducing another property. I think that `-DskipTests` is a good fit in our release process.

Reply:
@jmfinelli no, `-DskipTests` means compile tests but don't run them. So you are not correct with your assumption.

you should include the test module if there is not -Prelease or -PnexusStagingDeployment, this is why it would be better to have property flags like -Drelease and -DreleaseStaging that can be negated to include test if none of them is specified.